### ISSUE TITLE
Bash files in examples to rust-script?

### ISSUE THREAD

**Issue by Type1J** [chinedufn/percy#150]
I have people with Windows, macOS, and Linux machines. The Windows machines don't have bash, but they all have, or can get (through cargo), rust-script. Can the scripts be changed to use rust-script instead?

**Comment by Type1J** [comment]
...or, even better, could it be worked into `build.rs` somehow?

**Comment by chinedufn** [comment]
Hey there.

The scripts can't be replaced in favor of build scripts because some of the scripts start servers, which you wouldn't want a build script to do.

I'm not too familiar with Windows, but I agree that we want something that works for everyone.

Perhaps we can start by adding scripts called `script-name.windows.file-extension` that can be run on Windows.

Would that work for you?

**Comment by Type1J** [comment]
Yes, `.cmd` files in addition to `.sh` files that do the same thing would work fine.

I was thinking rust-script because its `.crs` files are just Rust with a comment at the top to say what cargo deps it needs, and it's directly executable (on Linux or macOS, you can `chmod 755 run_something.crs`, or register `.crs` on Windows to be run by `rust-script`) after a `cargo install rust-script` on any platform.

**Comment by chinedufn** [comment]
Got it thanks for explaining. Actually, I'd prefer to avoid needing one extra thing for people to install before they can run the examples.

**Comment by kotx** [comment]
PowerShell might also be considered an alternative to cmd.
